### ISSUE TITLE
serverFill.reservedSlots does not work

### ISSUE THREAD

**Issue by guidojw** [blake-mealey/mantle#97]
Hi! Cool project and I love seeing where this is going.

I have stumbled across a small bug, when I set serverFill.reservedSlots like:
```yml
maxPlayerCount: 16
serverFill:
  reservedSlots: 4
```
the actually set value is incorrect:
![image](https://user-images.githubusercontent.com/35309288/146715411-fb004a96-ed0a-4fd8-a28c-48104c9ac683.png)

Values in `.mantle-state.yml` seem to be correct:
```yml
maxPlayerCount: 16
...
socialSlotType: Custom
customSocialSlotCount: 4
```

**Comment by blake-mealey** [comment]
Interesting bug, thanks for report. Will look into as soon as possible.

**Comment by blake-mealey** [comment]
This has been fixed in [v0.10.2](https://github.com/blake-mealey/mantle/releases/tag/v0.10.2)
